### PR TITLE
assets: bump invalid_inclusion_proof to 0.2 bundle

### PR DESCRIPTION
**Pull request by woodruffw**
See https://github.com/sigstore/sigstore-python/pull/790.

### CODE REVIEW

**Comment by woodruffw** [comment]
Just to copy the rationale here: my understanding of the bundle specification is that (1) inclusion proofs are only required starting with 0.2, and (2) that inclusion proofs in 0.1 bundles aren't required to have checkpoints. Consequently, clients vary in their handling of invalid inclusion proofs in 0.1 bundles: sigstore-python, for example, will perform an online lookup if the checkpoint is missing (as it is here).

File: test/assets/a.txt.invalid_inclusion_proof.sigstore [hunk 1]
```diff
@@ -1 +1 @@
...
...hhREJrU0VKNlQyazRkbG95YkRCaFNGWnBURzFPZG1KVE9YcGhWMlI2Q21SSE9YbGFVekZxWWpJMWJXSXpTblJaVnpWcVdsTTViR1ZJVW5sYVZ6RnNZa2hyZEZwSFJuVmFNbFo1WWpOV2VreFlRakZaYlhod1dYa3hkbUZYVW1vS1RGZEtiRmxYVG5aaWFUbG9XVE5TY0dJeU5YcE1NMG94WW01TmRrNVVWWHBOZW1Nd1RWUlJOVTU1T1doa1NGSnNZbGhDTUdONU9IaE5TVWRMUW1kdmNncENaMFZGUVdSYU5VRm5VVU5DU0hkRlpXZENORUZJV1VFelZEQjNZWE5pU0VWVVNtcEhValJqYlZkak0wRnhTa3RZY21wbFVFc3pMMmcwY0hsblF6aHdDamR2TkVGQlFVZEtVM1JIVkVOM1FVRkNRVTFCVW5wQ1JrRnBRa05CTkdwYVVWQTBRM2ROYVZkdlpWTTNWMDFYTkRaUmEwazBaVGRQYzA1SU0zbFdhR1lLTlhka1FuWm5TV2hCVUVwWmVHUnphVGxPY1U5WVZscHpSVlYwUTNWd09HMHhiUzh5ZWtjek9VWlVSMnhuUlRCTmIzSkVSazFCYjBkRFEzRkhVMDAwT1FwQ1FVMUVRVEpuUVUxSFZVTk5SVmxYVW5kSk5WRktaVTkzVGtOMVZqUjBibG93YmpWUlRteFZiRkF3UW5SWU5WWXlXbFJSVEhGalVXSlhkRzVsUXpkMENreHdkR2xaWjNJd1dqWXlWVVJSU1hoQlR6WkpkRmhCU0N0ellscGpjMkpxTURoNGNqTkhRWEJOTm1ocWRubFVRV3d6T1hCVE0xa3pjMXAzUVhvNGJHWUtVVVJJVGt3MFpVRk1SVzh4YUdWQldWWm5QVDBLTFMwdExTMUZUa1FnUTBWU1ZFbEdTVU5CVkVVdExTMHRMUW89In19fX0="}]}, "messageSignature": {"messageDigest": {"algorithm": "SHA2_256", "digest": "oM/HEnHW4njlfNMy/5V8P3BD/do1TEy7GQow1W76Ab8="}, "signature": "MEQCIFOpaXKWvvBDwThDjTHX7tFF8liRoSxLZIsSeoUM/6D4AiBxV9/RnTMMw1t6nniX0rCuwrf8Vh+feLFu99m4ir+3yA=="}}
```
Comment (loosebazooka) [inline test/assets/a.txt.invalid_inclusion_proof.sigstore:1]: so this is the only change 0.1 -> 0.2?

**Review by steiza — APPROVED** [review]
Makes sense to me!

File: test/assets/a.txt.invalid_inclusion_proof.sigstore (hunk 1 above)
Comment (woodruffw) [reply test/assets/a.txt.invalid_inclusion_proof.sigstore:1]: Yep!

**Review by loosebazooka — APPROVED** [review]

**Pull request merged**